guides/common/modules/proc_viewing-vulnerabilities.adoc

```diff
@@ -0,0 +1,20 @@
+[id="viewing-vulnerabilities"]
+= Viewing vulnerabilities
+
+You can view Common Vulnerabilities and Exposures (CVE) from the {advisorengine}.
+You can examine how the CVEs affect your hosts.
+
+:FeatureName: {advisorengine}
+include::snip_technology-preview.adoc[]
+
+.Prerequisites
+* You have installed the {advisorengine}.
```

Comment:
@jeremylenz We will need to update the procedure for the full IoP experience

Reply:
Created [SAT-35876](https://issues.redhat.com/browse/SAT-35876) and assigned to Brian.